cmake/s1.cmake

```diff
@@ -1,153 +0,0 @@
-#
-# Copyright (c) 2020 Nordic Semiconductor
-#
-# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
-#
-
-if (CONFIG_BUILD_S1_VARIANT AND
-    ((${CONFIG_S1_VARIANT_IMAGE_NAME} STREQUAL "app" AND NOT IMAGE_NAME)
-    OR
-    ("${CONFIG_S1_VARIANT_IMAGE_NAME}" STREQUAL "${IMAGE_NAME}")))
-# Create second executable for the second slot of the second stage
-# bootloader. This is done inside this file since it is non-trivial to add
-# executable targets outside the root CMakeLists.txt. The problem is that
-# most of the variables required for compiling/linking is only available
-# in the scope of this file.
-
-  set(link_variant_name s1)
-  set(link_variant ${link_variant_name}_)
-
-  configure_linker_script(
-    ${link_variant}linker_prebuilt.cmd
-    "-DLINK_INTO_${link_variant_name}"
```

Comment:
Where is the code that modifies which area of flash S1 is linked into ?

Reply:
https://github.com/SebastianBoe/fw-nrfconnect-zephyr/pull/1/commits/dc49c027b351ad581441cbc840304d21757d2298#diff-54920465c2b794413e2858cbade114efdb0b29c59e83d7e188ba423887a2977aR35